lib/index.js

```diff
@@ -64,9 +100,9 @@ export default class Pageres {
 	 * @api public
 	 */
 
-	async run() {
+	async run(): Promise<stream$Readable[]> { // TODO: any way to define that it returns `this.items` and use its type?
```

Comment:
Isn't Flow able to analyse that automatically and infer the return type? Not sure though but flow analysis is one of the selling points, hence the name.